Sources/NIOSSHServer/ExecHandler.swift

```diff
@@ -12,6 +12,8 @@
 //
 //===----------------------------------------------------------------------===//
 
+#if canImport(Foundation.Process)
```

Comment:
TIL that you can import check types and not only modules.

With that being said, why don't we use `@available` instead like we do in tests?

Reply:
Because that doesn't stop it trying to _compile_ the code. If `Foundation.Process` was _present_ on e.g. iOS but not _available_, then we could use `@available` here too, but it is simply not _present_ in the iOS SDK (and others), so we must use a compiler directive.

Reply:
Makes sense! Thanks